sdk-java/src/main/java/com/spotify/confidence/telemetry/TelemetryClientInterceptor.java

```diff
@@ -0,0 +1,34 @@
+package com.spotify.confidence.telemetry;
+
+import com.spotify.telemetry.v1.Monitoring;
+import io.grpc.*;
+import java.util.Base64;
+import javax.annotation.Nullable;
+
+public class TelemetryClientInterceptor implements ClientInterceptor {
+  public static final Metadata.Key<String> HEADER_KEY =
+      Metadata.Key.of("X-CONFIDENCE-TELEMETRY", Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
Add the header in binary instead and the header with `-bin` suffix.

Reply:
We decided not to do binary headers after all but rather Base64 string headers so the same server interceptor will work for both grpc and http.